module/VuFind/tests/unit-tests/src/VuFindTest/Search/Blender/ParamsTest.php

```diff
@@ -141,7 +141,7 @@ class ParamsTest extends \PHPUnit\Framework\TestCase
                             'Field' => 'creationdate',
                         ],
                         'EDS' => [
-                            'Field' => 'PublishDate',
+                            'Field' => 'PublicationDate',
```

Comment:
I think in theory you should be able to map this to whatever field name, but the EDS code [already has hard-coded logic for PublicationDate](https://github.com/vufind-org/vufind/blob/dev/module/VuFindSearch/src/VuFindSearch/Backend/EDS/SearchRequestModel.php#L192), which I'm depending on elsewhere in this PR.  So if that hard coding is going to be fixed (which is probably low priority) it should be a separate PR.

Reply:
Agreed, even if the API doesn't really offer meaningful alternatives in the way of date options (and I'm not sure whether or not it actually does), there might be some value in making the hard-coded value a little more abstract for readability and future flexibility. But as you say, that's a problem for a separate PR, and probably not the highest priority to worry about until we have a use case. :-)